server_rust/Makefile

```diff
@@ -1,3 +1,13 @@
+build_and_run:
```

Comment:
I have some upstream changes for this file in #4 and also restructuring the project so I would like to get that in first and then you can update the README and add a new Makefile at the root of the project

Reply:
perfect!